### ISSUE TITLE
Dark Mode Support

### ISSUE THREAD

**Issue by BayuAngora** [YoussefRaafatNasry/portfolYOU#8]
This theme is better if there's a dark mode button feature.

**Comment by YoussefRaafatNasry** [comment]
I will try to work on this soon,
Maybe help if you could?

**Comment by BayuAngora** [comment]
This theme has simple dark mode button below. 
https://webjeda.com/hagura/

**Comment by mustaphayusuf** [comment]
even me really wished if this Page has dark mode.

**Comment by mustaphayusuf** [comment]
I really wished if this Page could have dark mode, It would good and nicer

**Comment by mustaphayusuf** [comment]
really wished if this Page could had dark mode, It would look good and nicer.

**Comment by udaygirish** [comment]
@YoussefRaafatNasry  Any update on the dark mode support, really want to see this clean web theme in dark mode!!

**Comment by BornToBeRoot** [comment]
Dark mode would be awesome. Anyway, thanks for this great project 👍

**Comment by BayuAngora** [comment]
> Dark mode would be awesome. Anyway, thanks for this great project 👍
> 
> Edit: Maybe you could use this: https://github.com/sandoche/Darkmode.js
> 
> I could embed it into the site, but the color of the text does not change:
> 
> ![image](https://user-images.githubusercontent.com/16019165/92285806-94e2fc00-ef05-11ea-8498-2e74b6b2a6d2.png)
> ![image](https://user-images.githubusercontent.com/16019165/92285823-9f9d9100-ef05-11ea-8d4b-c2aa69bb9e02.png)
> 
> Edit 2: The option `saveInCookies` should be set to false by default.

Text in dark mode should be configure on dark mode style part.

**Comment by aman-roy** [comment]
You can use 
`@media(prefers-color-scheme: dark) `

For more details check these changes in my own blog - https://github.com/aman-roy/amanroy.me/commit/59380cac20998ec8bc7e04f8cb944eccfe58c786

**Comment by udaygirish** [comment]
@aman-roy That's really great and it worked for me... Thanks a lot, man!!

**Comment by aman-roy** [comment]
@udaygirish Welcome!

**Comment by udaygirish** [comment]
@aman-roy I checked my web in several Operating systems don't know whether the problem is with mine or not. But on my mac and android, it is showing in dark but not in chrome browsers in Ubuntu OS. Have you checked on this ?? Just want to know how this is happening!! I have even checked some issues regarding this but still, it seems it is not resolved yet !! Any idea about this?

**Comment by aman-roy** [comment]
@udaygirish, I use Ubuntu 20.04 as my main driver. I also use chrome for browsing. Unfortunately, there is no support for dark mode in chrome. But don't worry I will help you out in that. But the solution is quite experimental. It has not been incorporated completely into chrome. 

## Solution
**Step 1: Open chrome.**
**Step 2: Write `chrome://flags/#enable-force-dark`into URL menu.** 
**Step 3: Enable the very first option.**
![Screenshot from 2020-09-13 06-46-27](https://user-images.githubusercontent.com/13317901/93008003-585e7280-f58d-11ea-87fd-0eb8e033d06b.png)

---
**Step 4: Relaunch.**
**Step 5: Go to http://udaygirish.github.io and check for yourself. It's done!**

**Comment by bitcoin4free** [comment]
would like a button on the page for dark mode

**Comment by YoussefRaafatNasry** [comment]
Dark theme is avilable now guys! 🥳 

First of all, I need to apologize for this taking more much much more than it should. And as we developers are addicted to dark themes, I decided it's time to implement this.

I tried the solution proposed by @BornToBeRoot, but the result's were as good as I expected, even after fixing the issues that were introduced by [Darkmode.js](https://darkmodejs.learn.uno/).

So I ended up using [ForEvolve/bootstrap-dark](https://github.com/ForEvolve/bootstrap-dark) and modyifying it a teeny tiny bit to match the theme style. I might have missed something or two, but I will fix them as soon as I have more time. Don't feel hesitated to report those bugs.
